lib/qesapdeployment.pm

```diff
@@ -413,4 +414,59 @@ sub qesap_ansible_cmd {
     assert_script_run($ansible_cmd);
 }
 
+
+=head3 qesap_ansible_script_output
+
+    Use Ansible to run a command remotely and get the stdout.
+    Command is executed with elevated privileges
+
+    qesap_ansible_script_output(cmd => 'crm status', provider => 'aws', host => 'vmhana01');
+
+    It uses playbook data/sles4sap/script_output.yaml
+
+    1. ansible-playbook run the playbook, remote cmd ar paramenter
+    2. the playbook executes the command and redirects the output to file, both remotely
+    3. the playbook download the file locally
+    4. the file is read and sored to be returned to the caller
+
+=over 3
+
+=item B<PROVIDER> - Cloud provider name, used to find the inventory
+
+=item B<CMD> - command to run remotely
+
+=item B<HOST> - filter hosts in the inventory
+
+=back
+=cut
+
+sub qesap_ansible_script_output {
+    my (%args) = @_;
+    croak 'Missing mandatory cmd argument' unless $args{cmd};
+    croak 'Missing mandatory host argument' unless $args{host};
+    my $inventory = qesap_get_inventory($args{provider});
+
+    if (script_run('test -e script_output.yaml') != 0) {
```

Comment:
No need to check against `!= 0` .. `if (script_run 'test -e script_output.yaml') {` is enough.